opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -81,7 +83,7 @@ def http_status_to_status_code(
     return StatusCode.ERROR
 
 
-def unwrap(obj: Union[object, str], attr: str):
+def unwrap(obj: object, attr: str):
```

Comment:
There's a type of redundancy here, since `str` is an `object`.

But on the first line of this function, the type checker is able to infer if it's a `str` or not.

Reply:
The union has been added for humans more than for type checkers